packages/cluster-api-provider-vsphere/bundle/config/upstream/metadata.yaml

```diff
@@ -36,3 +36,6 @@ releaseSeries:
   - major: 1
     minor: 4
     contract: v1beta1
+  - major: 1
+    minor: 5
+    contract: v1beta1
```

Comment:
Remove this line.

Reply:
This PR introduces a new minor version and hence the new entry.